.knip.json

```diff
@@ -1,27 +1,16 @@
 {
-  "ignore": [
-    ".github/**",
-    "**/*/fuels.config.*",
-    "**/*/.vitepress/*",
-    "/apps/docs/*",
-    "/packages/abi-typegen/test/**",
-    "templates/**",
-    "/apps/docs/**/*.test.ts",
-    "apps/create-fuels-counter-guide/**",
-    "apps/docs-api/**"
+  "entry": [
+    "apps/docs/src/versions.data.ts",
+    "apps/docs/src/guide/types/snippets/numbers/for-u8-u16-and-u32-2.ts"
   ],
+  "ignore": [".github/**"],
   "ignoreDependencies": [
-    "autocannon",
     "bun",
-    "@/sway-api/*",
-    "@fuel-ts/*",
-    "@fuels/connectors",
-    "@internal/fuel-core",
-    "@internal/forc",
-    "@types/jest",
+    "fuels",
+    "bun",
     "@types/rimraf",
-    "@graphql-codegen*",
-    "graphql-tag",
+    "@fuel-ts/abi-typegen",
```

Comment:
This is used as a command explicitly in [building the recipes](https://github.com/FuelLabs/fuels-ts/blob/master/packages/recipes/scripts/build-recipes.ts#L5) so I found creating a monkey patch for it a bit tricky